packages/p2p/src/utils/local-storage.js

```diff
@@ -0,0 +1,96 @@
+import ObjectUtils from '@deriv/shared/utils/object';
```

Comment:
You can directly import the functions you need to use as per #1386. The file path `@deriv/shared/utils` is no longer valid.
```suggestion
import { isEmptyObject } from '@deriv/shared';
```